### PR TITLE
typo: fix link

### PR DESCRIPTION
**Pull request by gasp**

### CODE REVIEW

File: src/docs/documentation/references/creating-plugins.mdx [hunk 1]
```diff
@@ -70,7 +70,8 @@ A set of preconfigured webpack config plugins
 exports.onCreateWebpackConfig = ({
  stage, getConfig, rules, loaders, actions
 }) => {
-  actions.setWebpackConfig({
```
Comment (renatobenks) [inline src/docs/documentation/references/creating-plugins.mdx:73]: hey, I think you're introducing another typo here, can you get this back?

File: src/docs/documentation/references/creating-plugins.mdx [hunk 2]
```diff
@@ -31,7 +31,7 @@ Use to modify or create custom project configurations.
 
 #### Params
 
-- **config:** Default [configuration object](http://localhost:3000/docs/project-configuration)
+- **config:** Default [configuration object](https://docz.site/docs/project-configuration)
```
Comment (renatobenks) [inline src/docs/documentation/references/creating-plugins.mdx:34]: awesome, thanks!

**Review by renatobenks — CHANGES_REQUESTED** [review]

File: src/docs/documentation/references/creating-plugins.mdx (hunk 1 above)
Comment (gasp) [reply src/docs/documentation/references/creating-plugins.mdx]: sorry about  that, it's fixed now!

**Review by renatobenks — APPROVED** [review]

**Comment by renatobenks** [comment]
thanks for contributing @gasp

**Pull request merged**